pkg/cluster/internal/delete/delete.go

```diff
@@ -33,16 +33,17 @@ func Cluster(logger log.Logger, p providers.Provider, name, explicitKubeconfigPa
 		return errors.Wrap(err, "error listing nodes")
 	}
 
+	logger.V(0).Infof("Removing context %q from kubeconfig ...", name)
```

Comment:
The call below may actually do nothing. We should only log more info when we actually did something.

Reply:
I've moved this into an else clause.